.github/workflows/workflow_test.yaml

```diff
@@ -62,7 +63,6 @@ jobs:
       - dockerfile
       - integration
       - integration-rock
-      - publish
```

Comment:
What is the reason for this removal please?

Reply:
When running in a PR from a forked repository, the `publish` job will be skipped. That's why it's removed here.

Reply:
Why is it skipped? Is it because the workflow doesn't have access to the required secrets? We have an open PR that discusses this topic and the best we have come up with so far is that PRs from forks should be merged into a separate branch after initial review so that the full test suit can be run: https://github.com/canonical/is-charms-contributing-guide/pull/52

Reply:
If we don't execute all the checks then any PR from a fork could introduce breaking changes to the parts of `operator-workflows` that are not tested which results in our workflows breaking in repositories that rely on it working

Reply:
Actually, I am not a fan of creating a new branch and PR ourself every time an outside contributor wants to contribute.

I think another approach for this problem is to use special comments to trigger actions for PR initiated by outside contributors. You can find an example here: https://github.com/orgs/community/discussions/25389

But implementing that can not be done very quickly, for now, we can just skip the check since this PR https://github.com/canonical/operator-workflows/pull/107 is waiting on use.

Reply:
As discussed, let's change the check to allow for skipped in the case of forks and we'll add a story to the backlog with the comment to trigger a workflow run

Reply:
I just add the `publish` workflow back to the check dependencies.